modules/UI.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import { BStorage } from "./BStorage.js";
```

Comment:
- [ ] Please don't disable **eslint** because linter checks for those errors which can break your application. :blush: